Adyen/Core/Core Protocols/PartialPaymentComponent.swift

```diff
@@ -7,14 +7,12 @@
 import Foundation
 
 /// Any component that provides partial payments.
-/// :nodoc:
+@_spi(AdyenInternal)
 public protocol PartialPaymentComponent: PaymentComponent {
```

Comment:
This should not be used by merchant?
How can they handle giftcard flow outside of our DropIn ?